its/src/test/java/com/sonar/maven/it/suite/JavaTest.java

```diff
@@ -120,6 +120,24 @@ void setJavaVersionProperties() throws IOException {
       entry("sonar.java.target", "1.8"));
   }
 
+  @Test
+  void extractJdkHomeFromJre() throws IOException {
```

Comment:
We will have a ticket to trace the history of this change, but it might help us in the future to know what kind of assertion we are validating. I suggest changing the method's name to `sonar_java_jdk_home_defaults_to_maven_home`.

Reply:
We might want to add a test where the `sonar.java.jdkHome` is configured in the project and should not be overridden

Reply:
the cases where the jdk is configured are already tested